clippy_lints/src/wildcard_imports.rs

```diff
@@ -100,13 +101,15 @@ declare_clippy_lint! {
 pub struct WildcardImports {
     warn_on_all: bool,
     test_modules_deep: u32,
+    ignored_segments: Vec<String>,
```

Comment:
Let's store this as an `FxHashSet<String>` rather than creating two hashsets each time

`segments.iter().any(|seg| ignored.contains(seg))` is roughly equivalent to what `.intersection(..).next().is_some()` does under the hood